.goreleaser.yml

```
@@ -0,0 +1,4 @@
+builds:
+  - ldflags:
+      - -X github.com/elastic/elastic-package/internal/version.CommitHash={{.ShortCommit}}
+      - -X github.com/elastic/elastic-package/internal/version.BuildTime={{.Timestamp}}
```

Users are going to be probably more interested about the version itself :slightly_smiling_face: Add the tag to the output of `elastic-package version` too, so it displays something like this?
```
$ goreleaser build --skip-validate --single-target
$ ./dist/elastic-package_darwin_amd64/elastic-package version
elastic-package has been installed.
elastic-package v0.9.1 version-hash e59a09c (build time: 2021-07-16T13:11:19+02:00)
```
```suggestion
      - -X github.com/elastic/elastic-package/internal/version.BuildTime={{.Timestamp}}
      - -X github.com/elastic/elastic-package/internal/version.Tag={{.Tag}}
```

---

I want to do this in the next follow-up (just after this one :) ) as it requires a bit of coding. With this PR I want to fix first what's still broken.

If you prefer I can try to modify version's logic in this PR (both options work for me).

---

SGTM to do it in a follow up.